server/routes/reporting/standard-report/definitions.js

```diff
@@ -367,6 +368,47 @@ const { dailyUtilisationDAO, dailyUtilisationJurorsDAO } = require('../../../obj
           totals: true,
         },
       },
+      'prepare-monthly-utilisation': {
+        title: 'Monthly wastage and utilisation report',
+        headings: [
+          'dateFrom',
+          'reportDate',
+          'dateTo',
+          'reportTime',
+          '',
+          'courtName',
+        ],
+        bespokeReport: {
+          dao: async(req) => await generateMonthlyUtilisationDAO.get(
+            req,
+            req.session.authentication.locCode,
+            req.params.filter,
+          ),
+          body: true,
+        },
+        unsortable: true,
+        exportLabel: 'Export raw data',
+      },
+      'view-monthly-utilisation': {
+        title: 'View monthly wastage and utilisation report',
+        headings: [
+          'courtName',
+          'reportDate',
+          '',
+          'reportTime',
+        ],
+        bespokeReport: {
+          dao: async(req) => await viewMonthlyUtilisationDAO.get(
```

Comment:
update to `dao: (req) => viewMonthlyUtilisationDAO.get(`.... reason is that the first call always fails

Reply:
updated.